ui/static/ui/js/learning_resources.jsx

```diff
@@ -188,8 +299,10 @@ define('learning_resources', [
             });
 
           thiz.setState({
-            vocabulariesAndTerms: vocabulariesAndTerms
+            vocabulariesAndTerms: vocabulariesAndTerms,
+            selectedVocabulary: vocabulariesAndTerms[0].vocabulary
```

Comment:
This solves the proximate issue, avoiding accessing an attribute of an undefined object, but you raise a broader question, which we haven't dealt with, which is that the whole LR panel should not show this chooser UI if there are no vocabularies assigned to the resource type.

```
          if (vocabulariesAndTerms.length) {
            thiz.setState({
              selectedVocabulary: vocabulariesAndTerms[0].vocabulary
            });
          };
```